docs/tools/vdb_table/data/singlestoredb.json

```diff
@@ -50,7 +50,7 @@
   "multi_vec": {
     "support": "full",
     "source_url": "",
-    "comment": "Multiple vector columns can be created and queried"
+    "comment": "Multiple input vectors can be compared"
```

Comment:
this attribute refers to being able to:
1. store multiple vector fields per data point,
2. query against multiple vector fields in a single call

Is this supported in SingleStore? Could you provide a doc link for the same?

Reply:
2 is possible but 1 is not! Happy to leave this field as partial or not supported

Reply:
I'm a bit confused about how 2 could be supported without 1. Would be nice to see an example query.

I'm going to change this one to None for now, and merge the PR. Thanks!